q2cli/commands.py

```diff
@@ -535,19 +535,20 @@ def __call__(self, **kwargs):
 
     def _execute_action(self, action, arguments, recycle_pool=None,
                         cache=None):
-        if recycle_pool is None:
-            results = action(**arguments)
-            results = results._result()
-        else:
-            pool = cache.create_pool(key=recycle_pool, reuse=True)
-            with pool:
+        with cache:
```

Comment:
What if `cache` is `None`? Is a value always passed at this point?

Reply:
Yes that does not need to be a kwarg we will always have a cache passed in either the one they set or the default cache